CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+Version [1.1](https://github.com/dsg-bielefeld/slurk/tree/v1.1) (2019-26-03)
```

Comment:
Dates should be in ISO format (yyyy-mm-dd). The particular dates that occur here happen to be decodable unambiguously (on earth, with only 12 months), but others may not be... :-)

Reply:
I'd say we should confine ourselves to Earth first. The internet is currently not sufficiently developed on other planets to use Slurk there. 

Furthermore we should generally release between the 13th and 31st, so we are on the safe side.